spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkLocalScan.java

```diff
@@ -32,10 +33,14 @@ class SparkLocalScan implements LocalScan {
   private final StructType readSchema;
   private final InternalRow[] rows;
 
```

Comment:
nit: Shall we group all vars together? There is an empty line before filters now.

Reply:
removed